pages/spaces/study-space-demo.js

```diff
@@ -36,86 +35,75 @@ const theme = createMuiTheme({
+    <Container className="mt-10">
+      <Grid className="grid grid-cols-12">
+        <Grid
+          item
+          color={theme.palette.primary.dark}
+          className="flex flex-row  rounded-full space-x-5 col-span-4">
+          <Box>
             <Button
-              color="secondary"
               variant="outlined"
-              className="rounded-full  outline-none">
+              color="primary"
+              className="flex flex-row justify-start rounded-full  outline-none">
+              <CreateIcon />
               <Typography variant="body1" className="uppercase">
-                Quit
+                Session
               </Typography>
             </Button>
-          </Grid>
+          </Box>
+          <Box className="text-primary-dark">
+            <SettingsIcon />
+          </Box>
+        </Grid>
 
-          <Grid item className="flex justify-start items-end col-span-2 px-8 ">
-            <Box
-              color={theme.palette.primary.dark}
-              className="flex flex-row space-x-5 ">
-              <VideocamOffIcon />
-              <MicIcon />
-            </Box>
-          </Grid>
+        <Grid item className="col-span-4">
+          <Container className="mb-24">
+            <Timer />
+          </Container>
+        </Grid>
 
-          {/* MUI grid can be an item AND a container. */}
-          <Grid item container className="col-span-8">
-            <Container className="mb-24">
-              <Grid className="grid grid-cols-6 ml-5 space-x-5 space-y-5 ">
-                <div className="mt-5 ml-5">
-                  <User />
-                </div>
-                {Array.from(Array(11).keys()).map((key) => (
-                  <User key={key} />
-                ))}
-              </Grid>
-            </Container>
-          </Grid>
-          <Grid item className="flex justify-end items-end col-span-2 px-8 ">
-            <Box
-              color={theme.palette.primary.dark}
-              className="flex flex-row space-x-5 ">
-              <LibraryMusicIcon />
-              <PeopleIcon />
-              <ChatIcon />
-            </Box>
-          </Grid>
+        <Grid item className="flex flex-row justify-end items-start col-span-4">
+          <Button variant="outlined" className="rounded-full  outline-none">
+            <Typography variant="body1" className="uppercase">
+              Quit
+            </Typography>
+          </Button>
+        </Grid>
+
+        <Grid className="flex justify-start items-end col-span-2 px-8 ">
+          <Box
+            color={theme.palette.primary.dark}
+            className="flex flex-row space-x-5 ">
+            <VideocamOffIcon />
+            <MicIcon />
+          </Box>
+        </Grid>
+
+        {/* MUI grid can be an item AND a container. */}
+        <Grid className="col-span-8">
+          <Container className="mb-24">
+            <Grid className="grid grid-cols-6 ml-5 space-x-5 space-y-5 ">
```

Comment:
I think for the sake of consistency, we should define column widths in the props instead of classNames